prow/tide/README.md

```diff
@@ -0,0 +1,128 @@
+   the default method of merge as value. Valid options are `squash`, `rebase`, and `merge`.
+   Defaults to `merge`.
+* `target_url`: URL for tide status contexts.
+* `pr_status_base_url`: The base URL for the PR status page.
+* `max_goroutines`: The maximum number of goroutines spawned inside the component to
+   handle org/repo:branch pools. Defaults to 20. Needs to be a positive number.
+
+### Queries
+
+The `queries` field specifies a list of queries.
+Each query corresponds to a set of **open** PRs as candidates for merging.
+It can consist of the following dictionary of fields:
+
+* `repos`: List of queried repositories.
+* `labels`: List of labels any given PR from the `repos` must posses.
+* `missingLabels`: List of labels any given PR from the `repos` must not posses.
+* `excludedBranches`: List of branches that get excluded when quering the `repos`.
+* `includedBranches`: List of branches that get included when quering the `repos`.
+* `reviewApprovedRequired`: If set, each PR in the query must have review approved.
+
+Under the hood, a query constructed from the fields follows rules described in
+https://help.github.com/articles/searching-issues-and-pull-requests/
+
+**Important**: Each query must return a different set of PRs. No two queries are allowed to contain the same PR.
+
+Every PR in any of the pools that is not **mergeable** is discarded from processing.
+
+### Example
+
+```yaml
+tide:
+  merge_method:
+    kubeflow/community: squash
+  target_url: https://prow.k8s.io/tide.html
+  queries:
+  - repos:
+    - kubeflow/community
+    - kubeflow/examples
+    labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - needs-ok-to-test
+    - needs-rebase
+```
+
+**Explanation**: The component starts periodically querying all PRs in `github.com/kubeflow/community` and
+`github.com/kubeflow/examples` repositories that have `lgtm` and `approved` labels set
+and do not have `do-not-merge`, `do-not-merge/hold`, `do-not-merge/work-in-progress`, `needs-ok-to-test` and `needs-rebase` labels set.
+All PRs that conform to the criteria are processed and merged.
+The processing itself can include running jobs (e.g. tests) to verify the PRs are good to go.
+At the same time all commits in PRs from `github.com/kubeflow/community` repository are squashed before merging.
+
+## Presubmits
+
+Before a PR is merged, Tide runs a set of jobs configured in `presubmits` part of the `config.yaml` file to verify the PR can be merged.
+
+Semantic of individual fields of the `presubmits` is described in [prow/README.md#how-to-add-new-jobs](https://github.com/kubernetes/test-infra/blob/master/prow/README.md#how-to-add-new-jobs).
+
+The `presubmits` part consists of a set of `org/repo` based jobs. A PR is merged only if all jobs that are specified for PR's repository succeed (unless configured otherwise).
+
+### Example
```

Comment:
This section is outside the scope of Tide. I think you can delete it. The link you provided above about adding new jobs should be sufficient.

Reply:
Done